### ISSUE TITLE
Zenodo release licensing information

### ISSUE THREAD

**Issue by augustfly** [nanograv/PINT#1788]
Hi nanograv/PINT developers.

I am trying to reach the maintainer who issues the releases to Zenodo e.g., Version 1:

https://doi.org/10.5281/zenodo.11396241

On Zenodo this maintainer has the handle "JPGlaser". I'm not sure who that is, and my messages via the Zenodo messaging tool don't seem to be getting through. 

The issue is that the deposits to Zenodo are coming over with a wrong reuse license. The last deposit (version 1) was manually refined (quite nicely) by JPGlaser, but this license issue got overlooked. 

The problem as I see it is that the zenodo copy is being given a "CC-BY 4.0" license, which is totally invalid for PINT. It looks like PINT is under a 3 clause BSD + a rider for government contributions+ MIT submodule dual licensing. I think the best outcome is to put the entire Zenodo version under BSD 3 clause and add a note to the landing page with the full license information. I could help with that if you give me clear instructions on how to update Zenodo. 

I know you've added a CFF file to the repository, but right now that isn't ensuring this particular metadata is synchronized. I'm happy to try to figure out the best CFF settings to make this Zenodo deposit less intensive.

Thanks.

**Comment by JPGlaser** [comment]
Hello @augustfly ,
Apologies, I didn't even know that Zenodo had a messaging backend (I did just manage to locate it, though there isn't a notification icon to alert me that there is something. I might suggest that for later :sweat_smile: )

Anyway, I didn't notice the licensing mismatch, so that's my fault there. I went ahead and corrected it by adding the text found in the LICENSE.md file as a custom license with a descriptive title.

I'd love to work with you on getting the CFF file fixed up. When I did the latest import, I ended up doing a lot of clean-up. That said, we have an authorship repository for our papers that should be easy to pull data from and update the CFF file if I can get the appropriate syntax to ensure a good link.

~ Joe G.
